tethys_cli/gen_templates/settings

```diff
@@ -28,24 +28,24 @@ SECRET_KEY = '{{ secret_key }}'
 DEBUG = {% if production %}False{% else %}True{% endif %}
 
 {% if allowed_hosts %}
-ALLOWED_HOSTS = {{ allowed_hosts|safe }}
-{% else %}
-ALLOWED_HOSTS = [{% if allowed_host %}'{{ allowed_host }}'{% endif %}]
+ALLOWED_HOSTS = {{ allowed_hosts }}
 {% endif %}
```

Comment:
Do we need an `else` case on this? Is it possible for `allowed_hosts` not to be defined? What if they pass in `--allowed-hosts False` or `--allowed-hosts ''`?